src/plopp/backends/matplotlib/__init__.py

```diff
@@ -21,9 +21,13 @@ def image(self, *args, **kwargs):
         return ImageMpl(*args, **kwargs)
 
     def figure1d(self, *args, **kwargs):
-        from .figure import figure1d as fig1d_mpl
-        return fig1d_mpl(*args, **kwargs)
+        from .utils import is_interactive_backend
+        if is_interactive_backend():
+            from .interactive import InteractiveFig
+            return InteractiveFig(*args, **kwargs)
+        else:
+            from .static import StaticFig
+            return StaticFig(*args, **kwargs)
 
     def figure2d(self, *args, **kwargs):
-        from .figure import figure2d as fig2d_mpl
-        return fig2d_mpl(*args, **kwargs)
+        return self.figure1d(*args, **kwargs)
```

Comment:
This confused me at first. Would be better to have a common internal function that is called by both `figure1d` and `figure2d`?